src/components/toaster/toast.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { CSSTransition } from 'react-transition-group';
+import classNames from 'classnames';
+import Icon from '@folio/stripes-components/lib/Icon';
+
+import style from './style.css';
+
+function captialize(word) {
```

Comment:
`import capitalize from 'lodash/capitalize';`

Reply:
Ohh, didn't know we had lodash in our project.